packages/enhanced/src/schemas/container/ModuleFederationPlugin.ts

```diff
@@ -553,6 +553,10 @@ export default {
     runtime: {
       $ref: '#/definitions/EntryRuntime',
     },
+    asyncBoundary: {
```

Comment:
Should we update this option in "packages/enhanced/src/schemas/container/ModuleFederationPlugin.check.ts" ?

Reply:
ahh yes, i forgot to do the .ts file only did the schema file.